include/UnifiedCvo/utils/CvoPointCloud.hpp

```diff
@@ -147,15 +147,15 @@ namespace cvo {
       return positions;
     }
     Eigen::Vector3f at(unsigned int index) const;
-    const Eigen::Matrix<float, Eigen::Dynamic, Eigen::Dynamic> & labels() const { return labels_;}
-    Eigen::VectorXf label_at(unsigned int index) const { return labels_.row(index); }
+//    const Eigen::Matrix<float, Eigen::Dynamic, Eigen::Dynamic> & labels() const { return labels_;}
+    Eigen::VectorXf label_at(unsigned int index) const & { return Eigen::Map<const Eigen::VectorXf>(points_[index].label_distribution, NUM_CLASSES); }
```

Comment:
Just to make sure, do you think it is 
 Eigen::VectorXf label_at(unsigned int index) const & { return Eigen::Map<const Eigen::VectorXf>(points_[index].label_distribution, NUM_CLASSES); }

or 

const Eigen::VectorXf &  label_at(unsigned int index) const { return Eigen::Map<const Eigen::VectorXf>(points_[index].label_distribution, NUM_CLASSES); }

Reply:
Thanks for pointing out! I believe that's my mistake